in/BIH_StudyDescriptions_Gen3.tsv

```diff
@@ -0,0 +1,5135 @@
+StudyDescription	Modality	project_id	platform	frequency
+XR Chest AP or PA	{'CR', 'PR', 'DX', 'XR', 'CT', 'MR', 'CR,DX', nan}	{'NIHCC-CXR8', 'TCIA-covid-19-ar', 'IDC-IDC_acrin_nsclc_fdg_pet', 'TCIA-cmb-lca', 'MIDRC-Open-R1', 'MIDRC-Open-A1_PETAL_BLUECORAL', 'IDC-IDC_cmb_crc', 'MIDRC-Open-A1_PETAL_REDCORAL', 'MIDRC-TCIA-COVID-19-AR', 'MIDRC-Open-A1_SCCM_VIRUS', 'TCIA-cmb-mml', 'TCIA-midrc-ricord-1c', 'MIDRC-TCIA-RICORD_1c', 'TCIA-cmb-crc', 'TCIA-cptac-ccrcc', 'IDC-IDC_midrc_ricord_1c', 'IDC-IDC_cmb_mml', 'IDC-IDC_cmb_lca', 'TCIA-pseudo-phi-dicom-data', 'IDC-IDC_cptac_ccrcc', 'MIDRC-Open-A1', 'TCIA-acrin-nsclc-fdg-pet', 'AIMI-CheXpertPlus', 'IDC-IDC_pseudo_phi_dicom_data', 'IDC-IDC_covid_19_ar'}	{'IDC', 'TCIA', 'AIMI', 'NIHCC', 'MIDRC'}	428198
+NLST-LSS	{'SR', 'CT', 'SEG'}	{'IDC-IDC_nlst'}	{'IDC'}	48544
+NLST-ACRIN	{'SR', 'CT', 'SEG'}	{'IDC-IDC_nlst'}	{'IDC'}	24569
+Histopathology	{'SR', 'ANN', 'SEG', 'SM'}	{'IDC-IDC_tcga_read', 'IDC-IDC_tcga_lusc', 'IDC-IDC_cptac_pda', 'IDC-IDC_tcga_thym', 'IDC-IDC_tcga_ov', 'IDC-IDC_cmb_mel', 'IDC-IDC_tcga_prad', 'IDC-IDC_tcga_gbm', 'IDC-IDC_cptac_coad', 'IDC-IDC_tcga_skcm', 'IDC-IDC_cptac_luad', 'IDC-IDC_tcga_dlbc', 'IDC-IDC_tcga_stad', 'IDC-IDC_tcga_blca', 'IDC-IDC_cptac_ucec', 'IDC-IDC_cptac_gbm', 'IDC-IDC_tcga_hnsc', 'IDC-IDC_tcga_meso', 'IDC-IDC_rms_mutation_prediction', 'IDC-IDC_tcga_paad', 'IDC-IDC_tcga_lgg', 'IDC-IDC_cmb_crc', 'IDC-IDC_icdc_glioma', 'IDC-IDC_tcga_esca', 'IDC-IDC_tcga_lihc', 'IDC-IDC_tcga_chol', 'IDC-IDC_cmb_brca', 'IDC-IDC_ccdi_mci', 'IDC-IDC_tcga_luad', 'IDC-IDC_tcga_cesc', 'IDC-IDC_tcga_kirc', 'IDC-IDC_cmb_aml', 'IDC-IDC_tcga_sarc', 'IDC-IDC_cmb_gec', 'IDC-IDC_tcga_pcpg', 'IDC-IDC_cptac_lscc', 'IDC-IDC_cptac_hnscc', 'IDC-IDC_nlst', 'IDC-IDC_tcga_kich', 'IDC-IDC_tcga_uvm', 'IDC-IDC_gtex', 'IDC-IDC_cmb_pca', 'IDC-IDC_tcga_kirp', 'IDC-IDC_cptac_sar', 'IDC-IDC_cmb_lca', 'IDC-IDC_cptac_aml', 'IDC-IDC_tcga_acc', 'IDC-IDC_tcga_coad', 'IDC-IDC_tcga_thca', 'IDC-IDC_cptac_cm', 'IDC-IDC_tcga_tgct', 'IDC-IDC_tcga_ucs', 'IDC-IDC_cptac_ccrcc', 'IDC-IDC_cptac_brca', 'IDC-IDC_cmb_ov', 'IDC-IDC_tcga_ucec', 'IDC-IDC_cmb_mml', 'IDC-IDC_tcga_brca', 'IDC-IDC_cptac_ov'}	{'IDC'}	17045
+CHEST AP PORT	{'CR', 'DX', nan}	{'MIDRC-TCIA-COVID-19-NY-SBU', 'IDC-IDC_covid_19_ny_sbu', 'TCIA-covid-19-ny-sbu'}	{'IDC', 'TCIA', 'MIDRC'}	12732
+CHEST PORT 1 VIEW (RAD)-CS	{'CR', 'CR,DX', 'DX'}	{'MIDRC-Open-A1'}	{'MIDRC'}	7341
+MAMMO screening digital bilateral	{'MG', nan}	{'TCIA-breast-cancer-screening-dbt', 'IDC-IDC_breast_cancer_screening_dbt'}	{'IDC', 'TCIA'}	6752
+CHEST AP VIEWONLY	{'CR', 'DX', nan}	{'MIDRC-TCIA-COVID-19-NY-SBU', 'IDC-IDC_covid_19_ny_sbu', 'TCIA-covid-19-ny-sbu'}	{'IDC', 'TCIA', 'MIDRC'}	5297
```

Comment:
What is the meaning of `nan` in Modality?

Reply:
Some repositories do not specify a Modality for studies. I did not assign one, so they get "nan", which means there is no value.